nixos/modules/config/update-users-groups.pl

```diff
@@ -334,8 +334,11 @@ sub parseUser {
 sub allocSubUid {
     my ($name, @rest) = @_;
 
-    # TODO: No upper bounds?
-    my ($min, $max, $delta) = (100000, 100000 + 100 * 65536, 65536);
+    # The upper bound of 29000 users comes is derived from limits in
```

Comment:
The wording here is a bit off?

Reply:
Good catch. I reworked this comment a few times. I removed "comes" in my latest change which should clear up the verbiage. Thanks!